src/elements/Progressbar/style.scss

```diff
@@ -0,0 +1,22 @@
+:local {
+  @import '../../styles/config';
+  $progress-bar-height: 13px;
+  $bar-background-color: #0290d7;
+  $container-background-color: #e6e6e9;
+  $progress-bar-radius: 30px;
```

Comment:
you can put these in config.scss file as a global variable

Reply:
Sure.